ansible/roles/debops.phpipam/defaults/main.yml

```diff
@@ -10,240 +10,328 @@
+  php5: 'php5_phpipam'
+
+  error_pages:
+    '400': '/index.php?page=error&section=400'
+    '401': '/index.php?page=error&section=401'
+    '403': '/index.php?page=error&section=403'
+    '404': '/index.php?page=error&section=404'
+    '500': '/index.php?page=error&section=500'
+
+  location:
+    '/api': |
+      try_files $uri $uri/ /api/index.php?$args =404;
+
+    '/': |
+      rewrite ^/login/dashboard/$                 /dashboard/ redirect;
+      rewrite ^/logout/dashboard/$                /dashboard/ redirect;
+      rewrite ^/tools/search/(.*)/(.*)/(.*)/(.*)$ /index.php?page=tools&section=search&addresses=$1&subnets=$2&vlans=$3&ip=$4 last;
+      rewrite ^/(.*)/(.*)/(.*)/(.*)/(.*)/$        /index.php?page=$1&section=$2&subnetId=$3&sPage=$4&ipaddrid=$5 last;
+      rewrite ^/(.*)/(.*)/(.*)/(.*)/$             /index.php?page=$1&section=$2&subnetId=$3&sPage=$4 last;
+      rewrite ^/(.*)/(.*)/(.*)/$                  /index.php?page=$1&section=$2&subnetId=$3 last;
+      rewrite ^/(.*)/(.*)/$                       /index.php?page=$1&section=$2 last;
+      rewrite ^/(.*)/$                            /index.php?page=$1 last;
+      try_files $uri $uri/ $uri.html $uri.htm /index.html /index.htm =404;
+# ]]]
+
+# User, group, database configuration [[[
+# ---------------------------------------
+
+# .. envvar:: phpipam__user [[[
+#
+# System user for phpIPAM application
+phpipam__user: 'phpipam'
 
                                                                    # ]]]
-# .. envvar:: phpipam_group [[[
+# .. envvar:: phpipam__group [[[
 #
-# FIXME(yaml4rst): Describe what this variable is doing.
-phpipam_group: 'phpipam'
+# System group for phpIPAM application
+phpipam__group: 'phpipam'
 
                                                                    # ]]]
-# .. envvar:: phpipam_home [[[
+# .. envvar:: phpipam__home [[[
 #
-# FIXME(yaml4rst): Describe what this variable is doing.
-phpipam_home: '{{ (ansible_local.root.home
+# Home directory of the ``phpipam__user`` user.
+phpipam__home: '{{ (ansible_local.root.home
                    if (ansible_local|d() and ansible_local.root|d() and
                        ansible_local.root.home|d())
-                   else "/var/local") + "/" + phpipam_user }}'
+                   else "/var/local") + "/" + phpipam__user }}'
 
                                                                    # ]]]
-# .. envvar:: phpipam_database_user [[[
+# .. envvar:: phpipam__database_user [[[
 #
 # phpIPAM MySQL database configuration
-phpipam_database_user: 'phpipam'
+phpipam__database_user: 'phpipam'
 
                                                                    # ]]]
-# .. envvar:: phpipam_database_host [[[
+# .. envvar:: phpipam__database_host [[[
 #
-# FIXME(yaml4rst): Describe what this variable is doing.
-phpipam_database_host: 'localhost'
+# The hostname or IP address of the database server to use for the
+# phpIPAM database.
+phpipam__database_host: 'localhost'
```

Comment:
This variable should get the hostname of the MariaDB server via Ansible local facts; the location of the database apssword in `secret/` directory should also be updated to use the correct one created by `debops.mariadb` role. Check the [`debops.librenms`](https://github.com/debops/debops/blob/master/ansible/roles/debops.librenms/defaults/main.yml#L223-L244) role for a similar example.

Reply:
Done